lib/order_item.rb

```diff
@@ -0,0 +1,8 @@
+class Order_item
```

Comment:
Similar to my comment on the Cart class - I think you could merge a lot of these methods and group them up in a single class (Just from my notes from last week: Classes - describes what the object type will be [person/place/thing] and methods - do things [action/experience/condition]). I think for the Cart and Order item' classes , the methods belong together :)